backend/codesirius/authentication/models/verification.py

```
@@ -75,7 +76,9 @@ def save(self, *args, **kwargs):
         if self._regenerated:
             # If the verification code is regenerated, do not save it.
             # As the verification code is already saved in the database.
-            raise Exception("You cannot save a regenerated verification code.")
+            raise ValidationError(
+                {"detail": "You cannot save a regenerated verification code."}
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)